lab_2/main.py

```diff
@@ -100,7 +214,32 @@ def get_sparse_vector(original_text: list, language_profiles: dict) -> list or N
     :param original_text: any tokenized text
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
-    pass
+    if not (
+        isinstance(original_text, list)
+        and all(isinstance(s, str) for s in original_text)
+        and isinstance(language_profiles, dict)
+        and language_profiles != {}
+        and all(isinstance(s, str) for s in language_profiles)
+        and all(isinstance(d, dict) for d in language_profiles.values())
+        ):
+        return None
+    lang_profiles_mix = {}
+    for freq_dict in language_profiles.values():
+        for index, token in freq_dict.items():
+            if token > lang_profiles_mix.get(index, 0):
+                lang_profiles_mix[index] = token
+    lang_features = get_language_features(language_profiles)
+    text_vector_standard = []
+    for feature in lang_features:
+        if feature in original_text:
+            text_vector_standard.append(lang_profiles_mix[feature])
```

Comment:
get

Reply:
I've changed this part of code.
However, I haven't used get(), because a feature must be a member of 2 lists (lang_features and original_text). 'List' objects have no attribute 'get'. That is why 'if' is used in the changed code.

Reply:
dictionaries

Reply:
changed my code a bit